omniledger/service/heartbeat.go

```diff
@@ -0,0 +1,115 @@
+package service
+
+import (
+	"errors"
+	"sync"
+	"time"
+)
+
+type heartbeat struct {
```

Comment:
A 3-5 line comment here explaining the general concept, and what problem it solves for us would be good here. (But no other comments needed in this file, it's clear enough other than "why does it exist?")